practice/bio/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Hekate V. Nichols</title>
+     <link rel="stylesheet" href="site.css">
+  </head>
+  <body>
+    <header>
+            <h1>Mx. Hekate V. Nichols</h1>
+    </header>
+      <img src="https://avatars2.githubusercontent.com/u/17284573?v=3&s=460">
+    <blockquote>
+      "What you see depends on where you stand."
+    </blockquote>
+    <section>
+      <article>
+        An ambitious production manager for a theatre company that she built
+        with local artists, who is looking to break into the programming world.
+      </article>
```

Comment:
I think the lowest level of text containment should be paragraphs `p`. Having an article in a section somehow implies that that second layer is a complex layer.